src/theory/fp/theory_fp.cpp

```diff
@@ -801,11 +802,13 @@ void TheoryFp::convertAndEquateTerm(TNode node) {
 #ifdef SYMFPUPROPISBOOL
       handleLemma(nm->mkNode(kind::EQUAL, node, converted));
```

Comment:
What about this case?

Reply:
I wasn't sure about the code that is under this flag.  There is another part of the code under this flag that appears to use a different interface for outputting lemmas.  @aniemetz  what is the status of this `SYMFPUPROPISBOOL`?  Are we planning to maintain this?

Reply:
That's a very good question. @martin-cs ?